src/garage/envs/grid_world_env.py

```diff
@@ -128,16 +238,5 @@ def get_possible_next_states(self, state, action):
         else:
             return [(next_state, 1.)]
 
-    @property
-    def action_space(self):
-        return gym.spaces.Discrete(4)
-
-    @property
-    def observation_space(self):
-        return gym.spaces.Discrete(self.n_row * self.n_col)
-
-    def render(self, mode='human'):
-        pass
-
     def log_diagnostics(self, paths):
```

Comment:
please delete this

Reply:
All abstract methods need to be overridden so I guess it has to stay:(

Reply:
which abstract class is requiring the method?

Reply:
`Environment`

Reply:
why does environment have `log_diagnostics`? Please remove.

Reply:
I thought you meant `render` lol. Will do.